src/util/updates.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+import updateNotifier from 'update-notifier';
```

Comment:
The key to testing this function is to stub out the `updateNotifier` since we can trust it works as documented. The goal of the tests will be to make sure that web-ext is configuring the notifier correctly.

First, import the original package as a default that can be customized:
````javascript
import defaultUpdateNotifier from 'update-notifier';
````
Next, use [dependency injection](https://en.wikipedia.org/wiki/Dependency_injection) so that it can be replaced:
````javascript
export function checkForAutomaticUpdates(
  {
    name,
    version,
    updateCheckInterval,
    updateNotifier = defaultUpdateNotifier,
  }: checkForAutomaticUpdatesParams
) {
  // ...
  updateNotifier();
}
````
This allows you to test it with a stub:
````javascript
const updateNotifier = sinon.stub();
checkForAutomaticUpdates({name, version, updateNotifier});

assert.equal(updateNotifier.firstCall.args[0].pkg.name, 'web-ext');
````